salt/utils/boto3_elasticsearch.py

```diff
@@ -1,130 +0,0 @@
-"""
-Botocore waiters for elasticsearch that are not present in boto3+botocore (yet).
-
-:codeauthor: Herbert Buurman <herbert.buurman@ogd.nl>
-:depends: boto3
-"""
-
-import salt.utils.versions
-from salt.exceptions import SaltInvocationError
-
-try:
-    import botocore.waiter
-except ImportError:
-    pass
-
-
-WAITER_CONFIGS = {
-    "ESDomainAvailable": {
-        "delay": 60,
-        "operation": "DescribeElasticsearchDomainConfig",
-        "maxAttempts": 60,
-        "acceptors": [
-            {
-                "expected": "Active",
-                "matcher": "path",
-                "state": "success",
-                "argument": "DomainConfig.ElasticsearchClusterConfig.Status.State",
-            },
-            {
-                "expected": True,
-                "matcher": "pathAny",
-                "state": "failure",
-                "argument": "DomainConfig.*.Status.PendingDeletion",
-            },
-        ],
-    },
-    "ESUpgradeFinished": {
-        "delay": 60,
-        "operation": "DescribeElasticsearchDomain",
-        "maxAttempts": 60,
-        "acceptors": [
-            {
-                "expected": False,
-                "matcher": "path",
-                "state": "success",
-                "argument": "DomainStatus.UpgradeProcessing",
-            }
-        ],
-    },
-    "ESDomainDeleted": {
-        "delay": 30,
-        "operation": "DescribeElasticsearchDomain",
-        "maxAttempts": 60,
-        "acceptors": [
-            {
-                "expected": True,
-                "matcher": "path",
-                "state": "retry",
-                "argument": "DomainStatus.Deleted",
-            },
-            {
-                "expected": False,
-                "matcher": "path",
-                "state": "failure",
-                "argument": "DomainStatus.Processing",
-            },
-            {
-                "expected": "ResourceNotFoundException",
-                "matcher": "error",
-                "state": "success",
-            },
-        ],
-    },
-    "ESDomainCreated": {
-        "delay": 30,
-        "operation": "DescribeElasticsearchDomain",
-        "maxAttempts": 60,
-        "acceptors": [
-            {
-                "expected": True,
-                "matcher": "path",
-                "state": "success",
-                "argument": "DomainStatus.Created",
-            }
-        ],
-    },
-}
-
-
-def __virtual__():
-    """
-    Only load if botocore libraries exist.
-    """
-    return salt.utils.versions.check_boto_reqs(check_boto=False)
-
-
-@salt.utils.decorators.is_deprecated(globals(), "Sulfur")
-def get_waiter(client, waiter=None, waiter_config=None):
```

Comment:
**question** looks like `salt/modules/boto3_elasticsearch.py` has a `conn.get_waiter("ESDomainAvailable")` (looks like it might be some kind of loader magic) - is that *not* using this util?

Reply:
@waynew for reference: https://github.com/saltstack/salt/pull/59882

Reply:
Ah :+1: that makes sense. I'm sure there's some magic out there that must be doing the actual getting the real thing instead of the utils one. Fair enough!